catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/IfThenBrickTest.java

```diff
@@ -72,39 +67,65 @@ public void setUp() throws Exception {
 
 	@Category({Cat.AppUi.class, Level.Smoke.class})
 	@Test
-	public void ifThenBrickDeleteTest() {
+	public void testCopy() {
```

Comment:
isnt all of this already tested in the adapter unit tests?

Reply:
yes, I did not add the test I just adapted it.

Reply:
we should consider creating a parameterized unit test that tests all composite bricks, but should I do it in this PR though? What about we leave it for now and improve in the next iteration?